synfig-studio/src/gui/duckmatic.h

```diff
@@ -202,6 +207,10 @@ class Duckmatic
 	bool alternative_mode_;
 	bool lock_animation_mode_;
 
+	bool has_guide_x = false;
+	bool has_guide_y = false;
```

Comment:
deleted: not used

Reply:
it is used in `snap_point_to_grid()` and set from `find_guide()`.

Reply:
done